stripper/ze_lotr_minas_tirith_p5.cfg

```diff
@@ -1,3 +1,211 @@
+;increase zombie ladder health to account for csgo weapons
+modify:
+{
+	match:
+	{
+		"classname" "func_physbox_multiplayer"
+		"targetname" "item_escalera_6"
+	}
+	replace:
+	{
+		"health" "3000"
+	}
+}
+
+;increase nazgul health to account for csgo weapons
+modify:
+{
+	match:
+	{
+		"classname" "func_physbox_multiplayer"
+		"targetname" "ph_item_nazgul_7"
+	}
+	replace:
+	{
+		"health" "2708"
+	}
+}
+
+;increase troll health to account for csgo weapons
+modify:
+{
+	match:
+	{
+		"classname" "func_physbox_multiplayer"
+		"targetname" "ph_troll_hp_2"
+	}
+	replace:
+	{
+		"health" "9500"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_physbox_multiplayer"
+		"targetname" "ph_troll_hp_23"
+	}
+	replace:
+	{
+		"health" "5000"
+	}
+}
```

Comment:
What criteria (if any) are you using to calculate these? Some HP values (item_escalera_6) have nearly doubled while others (ph_troll_hp_23) only go up by 20%.

I don't think there is any established numbers for balancing CS:S damage values to accommodate the CS:GO negev. Ultimately, it would be *very* hard to even do that accurately, considering it would depend on the amount of players even using the negev. And with CS2 so close anyways, trying to get into this kind of balancing now doesn't really seem like a good idea.

Reply:
This was a mistake on my end. My intention were meant to slightly buff it by a fixed percentage (~20%) but these changes were made separately. I started off with 2x increase but eventually decided on a slight 20% buff.

Reply:
The 2nd part of my comment still stands, I'm not sure how we can confidently say a 20% buff accurately accounts for the negev buff.